_tutorials/convert-html-site-to-jekyll.md

```diff
@@ -0,0 +1,481 @@
+
+```
+jekyll serve
+```
+
+When you build the site, you get a preview URL such as `http://127.0.0.1:4001/`. The site's files are built in the `_site` folder.
+
+This is a Jekyll site at the most basic level. Here's what is happening:
+
+* The `_config.yml` file contains settings that Jekyll uses as it processes your site. This basic config file tells Jekyll to convert Markdown to HTML using the [kramdown Markdown filter](https://rubygems.org/gems/kramdown/).
+* Jekyll looks for files with [front matter tags]({% link _docs/frontmatter.md %}) (the two sets of dashed lines `---` like those in `index.md`) and processes the files (populating site variables, rendering any [Liquid](https://shopify.github.io/liquid/), and converting Markdown to HTML).
+* Jekyll pushes the content from all pages and posts into the `{% raw %}{{ content }}{% endraw %}` tags in the layout specified (`default`) in the front matter tags.
+* The processed files get written as `.html` files in the `_site` directory.
+
+You can read more about how Jekyll processes the files in [Order of Interpretation](/tutorials/orderofinterpretation/).
+
+With this basic foundation of how a Jekyll site works, you can convert almost any HTML theme into a Jekyll site. The following sections will take you through a step-by-step tutorial on converting an HTML template into a Jekyll site.
+
+## 1. Create a template for your default layout
+
+Find your HTML theme and save it as a default layout. If you're converting or cloning an existing site, you can right-click the page and view the source code. For example, suppose you're cloning your company site to create a documentation site with the same branding. Or suppose you have a personal site that you built with HTML and now want to make it a Jekyll theme. Get the HTML source code for your site.
+
+Copy and paste the source code into a file called `default.html`. Put the `default.html` file inside a folder called `_layouts`. This will be the default layout template for your pages and posts &mdash; that is, each page or post will use this layout when Jekyll builds the site.
+
+Note that in looking for templates, you want the HTML output of the template. If the template has PHP tags or other dynamic scripts, these dynamic elements will need to be converted to HTML or stripped out, since Jekyll handles only static content.
+
+Open `default.html` into your browser locally to ensure the site looks and functions like it does online. You will likely need to adjust CSS, JS, and image paths so they work.
+
+For example, if the paths were relative on the site you copied, you'll need to either download the same assets into your Jekyll site or use absolute paths to the same assets in the cloud. Syntax such as `src="//` requires a prefix such as `src="http://` to work in your local browser.
+
+Jekyll provides some [filter tags](/docs/templates/) to prepend a site URL before path. For example, you could preface your stylesheet like this:
```

Comment:
Can we not mix two distinct entities here? `filters` and `tags` are two distinct functional entities in `Liquid`. 
```diff
- [filter tags](/docs/templates/)
+ [URL filters](/docs/templates/#filters)
```